cli/pluginerror.go

```diff
@@ -0,0 +1,17 @@
+package cli
```

Comment:
moove this error to the plugins package and call it: NotInstalledError / MissingError / NotFoundError. This will lead to a nice interface: plugins.NotFoundError.
PluginError does not describe what is the issue with the plugin, if you create this kind of error struct:

func NewNotFoundError(plugin string) error {

}
// NewPluginError returns a pluginError with the provided text
func NewPluginError(text string) error {
	return PluginError{
		s: text,
	}
}
 // PluginError ...
type PluginError struct {
	s string
}
 func (e PluginError) Error() string {
	return fmt.Sprintf("%s not installed", e.plugin)
}
....